src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,5 +1,6 @@
 Microsoft.CodeAnalysis.CommandLineSourceFile.CommandLineSourceFile(string path, bool isScript, bool isInputRedirected) -> void
 Microsoft.CodeAnalysis.CommandLineSourceFile.IsInputRedirected.get -> bool
+Microsoft.CodeAnalysis.Diagnostics.AnalyzerFileReference.Equals(Microsoft.CodeAnalysis.Diagnostics.AnalyzerFileReference other) -> bool
```

Comment:
Did we really want to make this public? If you can call the other one with a AnalyzerFileReference and get the same behavior, is it really needed?

Reply:
I guess it's not needed, just a common pattern for Equality/GHC.